docs/content/preview/yugabyte-platform/create-deployments/async-replication-platform.md

```diff
@@ -18,101 +18,165 @@ Within the concept of replication, universes are divided into the following cate
+    
+    To make the add table operation faster, it is recommended to add new tables to the replication as soon as they are created. When using YSQL, if a subset of tables of a database is already in replication in a configuration, the remaining tables in that database can only be added to the same configuration. Moreover, if adding a YSQL table requires bootstrapping, the other tables in the same database which are already in replication will be removed from replication. The replication will be set up again for all the tables in that database.
 
-- Pause the replication process (stop the traffic) by clicking **Pause Replication**. This is useful when performing maintenance. Paused replications can be resumed from the last checkpoint.
+- View detailed statistics on the asynchronous replication lag by selecting **Metrics**, as per the following illustration:
 
-- Delete the universe replication by clicking **Actions > Delete replication**.
+  ![Alert](/images/yp/asynch-replication-61.png)
 
 ## Set up bidirectional replication
 
 You can set up bidirectional replication using either the YugabyteDB Anywhere UI or API by creating two separate replication configurations. Under this scenario, a source universe of the first replication becomes the target universe of the second replication, and vice versa.
 
+Bootstrapping is skipped for tables on the target universe that are already in replication. It means that, while setting up bidirectional replication, bootstrapping can be done in only one direction, that of the first replication configuration. For example, when you set up replication for table  t1 from universe u1 to u2, it may go through bootstrapping. However, if  you set up replication in the reverse direction, from universe u2 to u1 for table t1, bootstrapping is skipped.
+
+## Limitations
+
+The current implementation of xCluster replication in YugabyteDB Anywhere has the following limitations:
+
+- The source and target universes must exist on the same instance of YugabyteDB Anywhere because data needs to be backed up data from the source universe and restored to the target universe.
+- The chosen storage configuration for bootstrapping must be accessible from both universes.
+- Active-active bidirectional replication is not supported because the backup or restore would wipe out the existing data. This means that bootstrapping can be done only if an xCluster configuration with reverse direction for a table does not exist. It is recommended to set up replication from your active universe to the passive target, and then set up replication for the target to the source universe. To restart a replication with bootstrap, the reverse replication must be deleted.
+- The tables with the same name (`database.schema_name.table_name` for YSQL and `keyspace.table_name` for YCQL) and schema must exist on both universes before xCluster replication can be set up.
+- An xCluster configuration can contain only YSQL or YCQL tables.
```

Comment:
nit: Remove this as it is mentioned above.

Reply:
@shahrooz1997 Do you mean I need to delete all four bullet points?

Reply:
No, just this line:
> An xCluster configuration can contain only YSQL or YCQL tables.

Reply:
Deleted